src/cache/cache_las.c

```diff
@@ -859,17 +866,59 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
 		 * should another thread remove the record before we do (not
 		 * expected for dropped trees), and the cursor remains
 		 * positioned in that case.
-		 *
+		 */
+		if (las_id >= cache->las_sweep_dropmin &&
+		    las_id <= cache->las_sweep_dropmax &&
+		    __bit_test(cache->las_sweep_dropmap,
+		    las_id - cache->las_sweep_dropmin)) {
+			WT_ERR(cursor->remove(cursor));
+			++remove_cnt;
+			continue;
+		}
+
+		/*
 		 * TODO it would also be good to remove entries in lookaside
 		 * from live files that have aged out.  If we track for each
 		 * entry whether it was the on-page value chosen by
 		 * reconciliation, we can safely remove entries from that point
 		 * on (for the given key) that are visible to all readers.
 		 */
```

Comment:
This comment isn't necessary any more, is it?

Reply:
fixed.